### PR TITLE
Rename the fieldList prop to sidebarComponent.

### LINKED ISSUE #28: App component relies on a prop named fieldList
It should be specified as a `defaultProps` instead, and should be renamed to "sidebarComponent" or something similar.

### PR DESCRIPTION
**Pull request by almet**
Fix #28 
r? @n1k0

### CODE REVIEW

File: formbuilder/routes.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ import Check from "./components/Check";
 
 const common = {
   notifications: NotificationContainer,
-  fieldList: FieldListContainer,
+  sidebarComponent: FieldListContainer,
```
Comment (n1k0) [inline formbuilder/routes.js]: I'd expect `FieldListContainer` to be renamed accordingly :)

Comment (almet) [inline formbuilder/routes.js]: The `FieldListContainer` container still remains a field list container, so I don't think we should change its name :)

Comment (n1k0) [inline formbuilder/routes.js]: I keep thinking it'd be sane to have a `SidebarContainer`.

Comment (almet) [inline formbuilder/routes.js]: I'm sorry, I don't understand the direction you want to take. Would you mind showing me what you have in mind with some code? (I just see another level of indirection without so much value so I'm probably missing your point)

Comment (n1k0) [inline formbuilder/routes.js]: r+ then ;)

**Pull request merged**